events/highlighter/highlighter.py

```diff
@@ -199,8 +199,10 @@ def switch_mode(mode):
 		return tokens
 
 	def highlight(func, theme="default"):
-		# Shotcuts
-		colors = Hl.Database.color_codes if theme == "default" else theme
+		# Shortcuts
+  
+		# the "shut up" type
+		colours: dict[str, str] = Hl.Database.color_codes # if theme == "default" else theme
```

Comment:
W variable rename from fake English to real English, respect.

Reply:
Stop reviewing on my repositories, nerd

Reply:
> Stop reviewing on my repositories, nerd

It's an open-source project, it's meant to be reviewed. Also, I was just complimenting Cobblestone, just like I complimented you in a later commit for using the correct spelling.